src/govuk/objects/_main-wrapper.scss

```diff
@@ -57,7 +57,7 @@
   // spacing without the need for the `l` modifier, but it is available in
   // instances where there may be empty elements that would be difficult to
   // remove.
-  .govuk-main-wrapper:first-child,
+  .govuk-main-wrapper--auto-spacing:first-child,
```

Comment:
Non-blocking: We could expand on the comment to say that this bit of code ensures the correct margin when there are no other elements like breadcrumbs or back link above the main container

Reply:
I think the comment is slightly out of date as it still talks about the `:first-child` as if it's always applied.

What about:

```scss
  // Using the `govuk-main-wrapper--auto-spacing` modifier should apply the
  // correct spacing depending on whether there are any elements (such as a back
  // link or breadcrumbs) before the `.govuk-main-wrapper` in the 
  // `govuk-width-container`.
  //
  // If you need to control the spacing manually, use the
  // `govuk-main-wrapper--l` modifier instead.
```

?